packages/http-client-java/generator/http-client-generator-mgmt/src/main/java/com/microsoft/typespec/http/client/generator/mgmt/util/FluentUtils.java

```diff
@@ -52,24 +44,12 @@ public class FluentUtils {
 
     private static final Logger LOGGER = new PluginLogger(FluentGen.getPluginInstance(), FluentUtils.class);
 
-    private static final Set<String> RESERVED_CLASS_NAMES = Collections
-        .unmodifiableSet(new HashSet<>(Arrays.asList(Response.class.getSimpleName(), Context.class.getSimpleName())));
+    private static final Set<String> RESERVED_CLASS_NAMES
+        = Set.of(ClassType.RESPONSE.getFullName(), ClassType.CONTEXT.getFullName());
```

Comment:
Here should use simple name, not full name.

```suggestion
    private static final Set<String> RESERVED_CLASS_NAMES
        = Set.of(ClassType.RESPONSE.getName(), ClassType.CONTEXT.getName());
```